libs/blocks/merch/README.MD

```diff
@@ -0,0 +1,14 @@
+# Overwriting default Commerce Checkout values:
```

Comment:
can you add this to the wiki on github instead? :)

Reply:
@vhargrave i have no permissions to edit wiki, who should i ask for access?

Reply:
I'd ask in the milo-dev channel on slack 👍